docs/source/user-guide/sql/aggregate_functions.md

```diff
@@ -0,0 +1,69 @@
+# Aggregate Functions
+
+Aggregate functions operate on a set of values to compute a single result.
+
+## General
+
+### min
```

Comment:
I think writing nice / proper documentation for SQL is a large undertaking.

For standard SQL functions, perhaps we can simply link to pre-existing documentation, such as: https://www.postgresql.org/docs/current/functions-aggregate.html and focus DataFusion documentation on what is different / specific to datafusion (such as`approx_percentile_cont`, `approx_median` and `approx_percentile_cont_with_weight`)?

Reply:
good point, I add the PostgreSQL link and only list what we support for General and Statistical functions. Approximate functions keep same detail documentations

[bcba321](https://github.com/apache/arrow-datafusion/pull/2082/commits/bcba321e0706f6ab1369b4ac730d13930f06bc6c)